sys/src/9/port/sysfile.c

```diff
@@ -1260,9 +1260,35 @@ sysmount(Ar0* ar0, ...)
 	dc = va_arg(list, int);
 	va_end(list);
 
+	if(dc == 0){
```

Comment:
I think we should not do this one part here? It should just be an error I think?

Reply:
Changed it to an error.